### ISSUE TITLE
empty string as default gives warning: Inline literal start-string without end-string.

### ISSUE THREAD

**Issue by maurerle** [sphinx-doc/sphinx-argparse#58]
```python
import argparse

def create_parser() -> argparse.ArgumentParser:
    parser = argparse.ArgumentParser(
        description="Command Line Interface",
    )
    parser.add_argument(
        "-c",
        "--case-study",
        help="the case study",
        default="",
        type=str,
    )
    return parser
```

Then adding:

```rst
.. argparse::
   :filename: ../mycli.py
   :func: create_parser
   :prog: mycli
```

This gives

```
:1: (WARNING/2) Inline literal start-string without end-string.                                                                                                              
:1: (WARNING/2) Inline literal start-string without end-string.
```
during `reading sources`

sphinx v6.2.1
sphinx-argparse v0.5.0
python 3.12.2

**Comment by AA-Turner** [comment]
Fixed in 9b7421eefbbd7835356d83ce10eae243e33dabe5, thank you Florian.

A

**Comment by AA-Turner** [comment]
I have released [sphinx-argparse v0.5.2](https://pypi.org/project/sphinx-argparse/0.5.2/) with a fix.

A

**Comment by maurerle** [comment]
Wonderful! thanks for the quick fix and release! :)
